src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/TrackingOptInPage.java

```diff
@@ -103,46 +103,98 @@ public void verifyTrackingRequestsNotSend(
     isTrackingRequestsNotSend(urlPatterns, networkTrafficInterceptor);
   }
 
+  public void verifyTrackingRequestsNotSendForAccepted(
+      List<String> urlPatterns, NetworkTrafficInterceptor networkTrafficInterceptor
+  ) {
+    isTrackingRequestsNotSendForAccepted(urlPatterns, networkTrafficInterceptor);
+  }
+
+  public void verifyTrackingRequestsSendForRejected(
+      List<String> urlPatterns, NetworkTrafficInterceptor networkTrafficInterceptor
+  ) {
+    isTrackingRequestsSendForRejected(urlPatterns, networkTrafficInterceptor);
+  }
+
   public void verifyTrackingRequestsSend(
       List<String> urlPatterns, NetworkTrafficInterceptor networkTrafficInterceptor
   ) {
     isTrackingRequestSend(urlPatterns, networkTrafficInterceptor);
   }
 
+  public void verifyTrackingRequestsSendOutsideEU(
+      List<String> urlPatterns, NetworkTrafficInterceptor networkTrafficInterceptor
+  ) {
+    isTrackingRequestSendOutsideEU(urlPatterns, networkTrafficInterceptor);
+  }
+
   public String urlOptInModalDisplayedOasis(Page page) {
     return urlBuilder.appendQueryStringToURL(page.getUrl(), MODAL_INSTANT_GLOBAL);
   }
 
   private void isTrackingRequestsNotSend(
-      List<String> elementsList, NetworkTrafficInterceptor networkTrafficInterceptor
+      List<String> listOfUrlsToVerify, NetworkTrafficInterceptor networkTrafficInterceptor
+  ) {
+    wait.forX(WAITING_TIME_FOR_ALL_REQUESTS);
+    isConsentCookieSetToRejected();
+    for (String url : listOfUrlsToVerify) {
+      Assertion.assertFalse(isSuccessfulResponseByUrlPattern(networkTrafficInterceptor, url),
+                            "Request to " + url + " services was found"
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Request to " 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)